Enable setting dict dir as command line option in the build process rather
than hard coding it into the header file
Author: Andreas Tille <tille@debian.org>

--- a/include/wn.h
+++ b/include/wn.h
@@ -27,7 +27,9 @@
 #define VRBIDXFILE	"%s\\sentidx.vrb"
 #define CNTLISTFILE     "%s\\cntlist.rev"
 #else
+#ifndef DICTDIR
 #define DICTDIR         "/dict"
+#endif
 #ifndef DEFAULTPATH
 #define DEFAULTPATH	"/usr/local/WordNet-3.0/dict"
 #endif
